src/EnergyPlus/HVACDXHeatPumpSystem.cc

```diff
@@ -128,8 +128,6 @@ namespace HVACDXHeatPumpSystem {
         std::string CompName; // Name of CoilSystem:Heating:DX object
         int DXSystemNum;      // Index to CoilSystem:Heating:DX object
         Real64 AirMassFlow;   // DX System air mass flow rate
-        int InletNodeNum;     // DX System inlet node number
-        int OutletNodeNum;    // DX System outlet node number
```

Comment:
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:131]:(style),[variableScope],The scope of the variable 'InletNodeNum' can be reduced.
[src/EnergyPlus/HVACDXHeatPumpSystem.cc:132]:(style),[variableScope],The scope of the variable 'OutletNodeNum' can be reduced.